app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -644,10 +647,45 @@ fun AppPresentationView(app: App) {
           overflow = TextOverflow.Ellipsis,
         )
       }
+      AppRating(rating = app.pRating)
     }
   }
 }
 
+@Composable
+fun AppRating(
+  modifier: Modifier = Modifier,
+  rating: Rating,
+) {
+  Row(
+    modifier = modifier
+      .wrapContentHeight
+        (
+        unbounded = true,
+        align = Alignment.Top
+      ),
+    verticalAlignment = Alignment.CenterVertically
+  ) {
+    Icon(
+      imageVector = getRatingStar(Palette.Black),
+      contentDescription = null,
+      modifier = Modifier
+        .padding(end = 2.dp)
+        .size(12.dp),
+    )
+    Text(
+      text = if (rating.avgRating == 0.0) {
+        "--"
+      } else {
+        TextFormatter.formatDecimal(rating.avgRating)
+      },
+      maxLines = 1,
+      style = AGTypography.InputsS,
```

Comment:
This appears to be a recent change, but in the design the text style is now InputsXS. This style probably needs to be added to the design system.